python/spacewalk/server/dbConfig.py

```diff
@@ -0,0 +1,64 @@
+#  pylint: disable=missing-module-docstring,invalid-name
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025 SUSE LLC
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+from . import rhnSQL
```

Comment:
We prefer not to use relative imports ([style guide](https://google.github.io/styleguide/pyguide.html#22-imports))

Reply:
Well, this . was introduced at python3 migration time by somebody of the python squad.
I think it has a reason. But I will try if the full import works